tests/Benchmark/Utf8FormatterBenchmark.cs

```diff
@@ -12,11 +12,11 @@ public class Utf8FormatterBenchmark
         private static readonly StatsDUtf8Formatter FormatterBuffer = new StatsDUtf8Formatter("hello.world", new NoOpTagsFormatter());
         private static readonly StatsDUtf8Formatter FormatterWithTagsBuffer = new StatsDUtf8Formatter("hello.world", TagsFormatter.DataDog);
 
-        private static readonly IDictionary<string, string?> EmptyTags = new Dictionary<string, string?>();
-        private static readonly IDictionary<string, string?> AnyValidTags = new Dictionary<string, string?>
+        private static readonly Dictionary<string, string?> EmptyTags = new Dictionary<string, string?>();
+        private static readonly Dictionary<string, string?> AnyValidTags = new Dictionary<string, string?>
         {
-            ["key"] = "value",
```

Comment:
Why change the syntax style?

Reply:
Don't know. I have changed from to KeyValuePair<string, string?>[], array of tuples and I didn't notice that I have changed at the end. Reverted!